cabal-testsuite/PackageTests/LinkerOptions/NonignoredConfigs/cabal.test.hs

```diff
@@ -71,6 +71,7 @@ main = cabalTest $ do
                     withDirectory ".." $ do
                         packageEnv <- (</> ("basic" ++ show idx ++ ".env")) . testWorkDir <$> getTestEnv
                         cabal "v2-install" $ ["--disable-deterministic", "--lib", "--package-env=" ++ packageEnv] ++ linkConfigFlags linking ++ ["basic"]
+                        cabal "v2-install" $ ["--disable-deterministic", "--lib", "--package-env=" ++ packageEnv] ++ linkConfigFlags linking ++ ["basic"]
```

Comment:
I think you need to tell it to reinstall here to avoid the validate error?

Reply:
Yes sorry @geekosaur, I pushed a work in progress experiment to see if it would fail before adding the flag for reinstalling.